phira/locales/en-US/respack.ftl

```diff
@@ -2,13 +2,13 @@
 label = RESPACK
 
 default = Default
-load-failed = Failed to load respack
+load-failed = Failed to load respacks.
```

Comment:
About the plural, can it be error when load multiple respacks, official?

Reply:
i was confused if it meant "error when importing a respack" or "error when loading imported respacks"

Reply:
I guess it's the respack the user trying to import, so maybe "load the respack" may sound better? I'm not very sure if this is grammarly ok here